src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,18 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
+    private static final User[] users = new User[] {
+            new User("bob@i.ua", "1234"),
+            new User("alice@i.ua", "1234")
+    };
+
     public boolean login(String email, String password) {
+        for (User user : users) {
+            if (user.getEmail().equals(email) && user.getPassword().equals(password)) {
+                return true;
+            }
+        }
         return false;
```

Comment:
I believe the desired solution here is to leverage `UserService`, which is already created and it's `findByEmail` method.